src/app/components/Modal/CharacterModal.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import moment from 'moment';
+import { useQuery } from '@apollo/client';
+import {
+  Modal,
+  Card,
+  Spin,
+} from 'antd';
+import { CHARACTERQUERY } from 'utils/graphql/queries';
+import { IModal } from 'utils/types/common';
+
+const CharacterModal : React.FC<IModal> = (props: IModal) => {
+  const {
+    isModalVisible,
+    handleOk,
+    handleCancel,
+    id,
+  } = props;
+  const { data: response } = useQuery(CHARACTERQUERY, {
+    variables: {
+      id,
+    },
+  });
+
+  return (
+    <Modal
+      visible={isModalVisible}
+      onOk={handleOk}
+      onCancel={handleCancel}
+    >
+      {response ? (
+        <Card
+          style={{ width: 240, marginLeft: 110 }}
+          cover={<img alt="example" src={response.character.image} className="modal-image" />}
+        >
+          <Card.Meta
+            style={{ marginBottom: '15px' }}
```

Comment:
there seems to be a lot of repetition of styling. what are the problems with having this repeated this many times and how can you DRY it?

Reply:
I think that it will affect the optimization of the project when we render the project.
So, To avoid this problem, I have to implement this style as a class and import it as a CSS file.